easyrsa3/easyrsa

```diff
@@ -298,20 +298,14 @@ Type the word '$value' to continue, or any other input to abort."
 	exit 9
 } # => confirm()
 
-# remove temp files
-clean_temp() {
+# remove temp files and do terminal cleanups
+cleanup() {
 	for f in "$EASYRSA_TEMP_CONF" "$EASYRSA_TEMP_EXT" "$EASYRSA_TEMP_FILE_2" "$EASYRSA_TEMP_FILE_3"
 	do	[ -f "$f" ] && rm "$f" 2>/dev/null
 	done
-} # => clean_temp()
-
-prog_exit() {
-	ESTAT=0
-	[ -n "$1" ] && ESTAT=$1
-	(stty echo 2>/dev/null) || set -o echo
+	(set +o echo 2>/dev/null && set -o echo 2>/dev/null) || stty echo
```

Comment:
I have tested Line 311 `(stty echo 2>/dev/null) || set -o echo` on W7 Enterprise 32bit and it works, so this specific change is not required.  See #276 for details.
Does this not work for you ?

Reply:
I could not reproduce the problem that required two 'set'

However, set -o subshell is needed for freebsd. If 'set -o echo' fails in a script, it exits. I know that stty will probably work there but I would keep the subshell.